evm-contracts/src/v0.7/interfaces/OracleInterface.sol

```diff
@@ -10,9 +10,18 @@ interface OracleInterface {
     uint256 expiration,
     bytes32 data
   ) external returns (bool);
+  function fulfillOracleRequest2(
```

Comment:
This changes the interface to not match with the previous solidity versions. I think we should leave the method where is, rename `OracleInterface2` to `OperatorInterface`, and make it inherit from the original `OracleInterface`.

Reply:
Noted. What about our other new functions:

- `operatorTransferAndCall`
- `distributeFunds`

These are better suited to `OperatorInterface` rather than `OracleInterface` IMO.